build/sbt-launch-lib.bash

```diff
@@ -39,7 +39,13 @@ dlog () {
 
 acquire_sbt_jar () {
   SBT_VERSION=`awk -F "=" '/sbt\.version/ {print $2}' ./project/build.properties`
-  URL1=https://repo1.maven.org/maven2/org/scala-sbt/sbt-launch/${SBT_VERSION}/sbt-launch-${SBT_VERSION}.jar
+  #
+  # TYPESAFE_MIRROR and DEFAULT_ARTIFACT_REPOSITORY env variables can be used to only fetch
+  # artifacts from internal repos only.
+  # Ex:
+  #   TYPESAFE_MIRROR=https://artifacts.internal.com/lightbend-dist
```

Comment:
`TYPESAFE_MIRROR` is only used at `build/mvn`
```
$ git grep TYPESAFE_MIRROR
build/mvn:    local TYPESAFE_MIRROR=${TYPESAFE_MIRROR:-https://downloads.lightbend.com}
build/mvn:      "${TYPESAFE_MIRROR}/zinc/${ZINC_VERSION}" \
build/mvn:  local TYPESAFE_MIRROR=${TYPESAFE_MIRROR:-https://downloads.lightbend.com}
build/mvn:    "${TYPESAFE_MIRROR}/scala/${scala_version}" \
```

Shall we skip to mention it here?